tensorrt_llm/_torch/auto_deploy/export/library/contiguous_check.py

```
@@ -0,0 +1,30 @@
+    This patch bypasses the complex Min/Max guard generation that occurs
+    during view/reshape operations with dynamic shapes. The underlying
+    view operations still validate tensor contiguity through other means.
+    """
+
+    def _apply_patch(self):
+        """Apply the contiguous check bypass patch."""
+        self.original_values["check_contiguous_sizes_strides"] = (
+            prims_common.check_contiguous_sizes_strides
+        )
+        prims_common.check_contiguous_sizes_strides = (
+            lambda sizes, strides, false_if_dde=False: True
```

Ruff (0.14.10)

Unused lambda argument: `sizes`

(ARG005)